### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/middleware.md`

### PR DESCRIPTION
**Pull request by lasr3**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]
Excellent, keep going with this kind of contribution.

**Comment by nazarepiedady** [comment]
@ceb10n and @cassiobotaro, could one of you do the kindness of reviewing this pull request?

File: docs/pt/docs/advanced/middleware.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+## Outros middlewares
+
+Há muitos outros middlewares ASGI.
+
+Por exemplo:
+
+* <a href="https://github.com/encode/uvicorn/blob/master/uvicorn/middleware/proxy_headers.py" class="external-link" target="_blank">Uvicorn's `ProxyHeadersMiddleware`</a>
+* <a href="https://github.com/florimondmanca/msgpack-asgi" class="external-link" target="_blank">MessagePack</a>
+
+To see other available middlewares check <a href="https://www.starlette.io/middleware/" class="external-link" target="_blank">Starlette's Middleware docs</a> and the <a href="https://github.com/florimondmanca/awesome-asgi" class="external-link" target="_blank">ASGI Awesome List</a>.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/middleware.md:96]: You've forgotten to remove this line 😄

**Review by ceb10n — CHANGES_REQUESTED** [review]
nice work @devluisrodrigues 

Just a line that you've forgotten to remove, and a quote in the admonition

File: docs/pt/docs/advanced/middleware.md (hunk 1 above)
Comment (devluisrodrigues) [reply docs/pt/docs/advanced/middleware.md:96]: oh haha, I'm sorry for that. I'll fix it!

**Comment by devluisrodrigues** [comment]
Hey @ceb10n and @nazarepiedady thank you for your feedback! I'll adress the changes and make a PR

**Comment by devluisrodrigues** [comment]
I just committed the changes. @ceb10n if you need anything else, feel free to ask!

**Review by ceb10n — APPROVED** [review]
🚀

**Comment by alejsdev** [comment]
Great, thanks! @devluisrodrigues :tada:

**Pull request merged**